### ISSUE TITLE
Update LightBDD.Fixie2 to LightBDD.Fixie3

### ISSUE THREAD

**Issue by Suremaker** [LightBDD/LightBDD#292]
#### Description
Upgrade LightBDD.Fixie2 to LightBDD.Fixie3 project

#### Progress

- [x] Create `LightBDD.Fixie3` project with fixie 3.x dependency
- [x] Remove `LightBDD.Fixie2` project and obsolete `LightBDD.Fixie2` package
- [x] Update examples
- [x] Update documentation (wiki)
- [x] Update VSIX templates
- [x] Ensure Scenario Ignore works
- [x] Ensure Category filtering works on execution

**Comment by Suremaker** [comment]
Blocked by https://github.com/fixie/fixie/issues/57#issuecomment-1614884438

**Comment by Suremaker** [comment]
Implemented LightBDD.Fixie3 project with the following changes in relation to LightBDD.Fixie2:
* Adding `class WithLightBddDiscovery: LightBddDiscoveryConvention { }` is no longer needed
* The discovery process supports running pre-selected categories by default. It is enough to call `dotnet fixie -- --category CatA --category CatB` to only run tests with category `CatA` or `CatB`

**Comment by Suremaker** [comment]
Released with LightBDD 3.7.0
